CITATION.cff

```diff
@@ -0,0 +1,17 @@
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.0.3
+message: If you use this software, please cite it as below.
+# FIXME title as repository name might not be the best name, please make human readable
+title: Singularity 2.5.2 - Linux application and environment containers for science
+doi: 10.5281/zenodo.1308868
```

Comment:
This is a per version citation thing isn't it? So for 2.6.0 we would need a different `doi` ID right?

Reply:
@jmstover the doi will be created automatically by Zenodo but yes the CITATION file would have to be updated manually.

Reply:
We can use the parent DOI in this case:

![screen shot 2018-07-31 at 1 52 35 pm](https://user-images.githubusercontent.com/4522799/43480480-0256bfd6-94c9-11e8-828a-4c45f4dd8d1c.png)